CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#9629](https://github.com/rubocop/rubocop/issues/9629): Add `AllowParenthesesInStringInterpolation` configuration to `Style/MethodCallWithArgsParentheses` to allow parenthesized calls in string interpolation. ([@gsamokovarov][])
```

Comment:
The changelog entries should now go to the `changelog` folder. (this file is auto-generated from them on release)

Reply:
Thanks, moved the entry to its own file. Will do so to my other PRs as well.